### PR TITLE
Release 1.8.0

### PR DESCRIPTION
**Pull request by minevala**
Release 1.8.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,55 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+#### Added
+- [Icons] New Icons: Icon set 12/2021
+- [Search Input] Support for onChange property to listen to input value changes
+- [Search Input] Support hiding search button
+- [Tabs] Enable customisation through theme prop
+- [Dropdown] Add icon support for multi-select variants of Select and Combobox components
+
+#### Fixed
+- [Buttons] Add active state to HDS buttons to add focus outline to Safari on click
```
Comment (VilleMiekkoja) [inline CHANGELOG.md:26]: This also applies to HDS Core

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,55 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### Core
+
+#### Added
+- [Icons] New Icons: Icon set 12/2021
+
+### React Components
+
+#### Added
+- [Icons] New Icons: Icon set 12/2021
```
Comment (VilleMiekkoja) [inline CHANGELOG.md:19]: I think this could be added to HDS Core also

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,55 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+  - Copy
+  - Restaurant
+  - Sitemap
+- [Modals and Notifications] Added new arrow positions for Tooltip to better match the implementation. New arrow symbols allow setting the arrow closer to the edge of the Tooltip.
+- [Modals and Notifications] Examples of all possible arrow positions
+
+**Changed**
+- Updated HDS Sketch libraries to Sketch 80.1
+- [Modals and Notifications] Slightly altered the symbol structure of the internal Tooltip symbols
+- [Modals and Notifications] Renamed Above and Below Tooltips to Top and Bottom to better match terms used in implementation
```
Comment (VilleMiekkoja) [inline CHANGELOG.md:48]: ... to better match terms used in **the** implementation

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,55 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- [Modals and Notifications] Swapped names of Left and Right Tooltips to better match terms used in implementation
```
Comment (VilleMiekkoja) [inline CHANGELOG.md:49]: .. used in **the** implementation

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,55 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### Documentation
+
+#### Added
+- New Icons: Icon set 12/2021
+- Example of custom themed Tabs
```
Comment (VilleMiekkoja) [inline CHANGELOG.md:55]: custom themed -> custom-themed

File: CHANGELOG.md (hunk 1 above)
Comment (minevala) [reply CHANGELOG.md:26]: - [ ] add fix to Core as well

File: CHANGELOG.md (hunk 2 above)
Comment (minevala) [reply CHANGELOG.md:19]: - [ ] add icons to Core as well

File: site/docs/about/new.mdx [hunk 6]
```diff
@@ -13,6 +13,18 @@ import Link from "../../src/components/Link";
     Here you will find summarized patch notes of major releases of HDS. For full patch notes for each release, please refer to the <Link href="https://github.com/City-of-Helsinki/helsinki-design-system/releases" external>GitHub releases</Link>.
 </LargeParagraph>
 
+<p><strong>1.8.0</strong> - <em>9.12.2021</em> - <Link href="https://github.com/City-of-Helsinki/helsinki-design-system/releases/tag/v1.8.0" external>Release notes</Link></p>
+
+- **Added**: New icons: Icon set 12/21
+- **Added**: Support for customising Tabs with theme
```
[the remark this answers is not in the document]
Comment (minevala) [reply site/docs/about/new.mdx:19]: - [ ] fix grammar

File: CHANGELOG.md (hunk 3 above)
Comment (minevala) [reply CHANGELOG.md:48]: - [ ] fix grammar

File: CHANGELOG.md (hunk 4 above)
Comment (minevala) [reply CHANGELOG.md:49]: - [ ]  fix grammar

File: CHANGELOG.md (hunk 5 above)
Comment (minevala) [reply CHANGELOG.md:55]: - [ ]  fix grammar

**Review by laurakarhu — APPROVED** [review]
Looking good, ready to ship it!

**Review by ronijaakkola — APPROVED** [review]

**Review by VilleMiekkoja — APPROVED** [review]
Nice 👍

**Pull request merged**